pairing_bot.go

```diff
@@ -255,6 +260,59 @@ func (pl *PairingLogic) endofbatch(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (pl *PairingLogic) checkin(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	numPairings, err := pl.pdb.GetTotalPairingsDuringLastWeek(ctx)
+
+	if err != nil {
+		log.Println("Total Pairings: ", numPairings)
+	}
+
+	recursersList, err := pl.rdb.GetAllUsers(ctx)
+	if err != nil {
+		log.Printf("Could not get list of recursers from DB: %s\n", err)
+	}
+
+	checkinMessage := getCheckinMessage(numPairings, len(recursersList))
+
+	botPassword, err := pl.adb.GetKey(ctx, "apiauth", "key")
+	if err != nil {
+		log.Println("Something weird happened trying to read the auth token from the database")
+	}
+
+	recurserEmail := "thecrxu@gmail.com"
+
+	err = pl.un.sendUserMessage(ctx, botPassword, recurserEmail, checkinMessage)
```

Comment:
In the future this will post to a Pairing Bot Topic in the checkins stream! Right now for testing purposes it just sends me a PM.